### PR TITLE
Meetup sept 2020

### PR DESCRIPTION
**Pull request by josisusan**

### CODE REVIEW

File: js/meetup/data.js [hunk 1]
```diff
@@ -33,7 +72,7 @@ const meetups = [
         title: 'Comments in Rails Migrations and Databases',
         description: null,
         speaker: 'Suman Awal',
-        twitter_username: 'awalsuman43',
```
Comment (josisusan) [inline js/meetup/data.js:36]: I removed this as there was some issue with the twitter profle pic.

File: js/meetup/data.js [hunk 2]
```diff
@@ -1,4 +1,43 @@
 const meetups = [
+  {
+    episode: 17,
```
Comment (zoras) [inline js/meetup/data.js:3]: This should be 19 as we have already done 2 remote meetups in may and june
https://www.meetup.com/Nepal-Ruby-Users-Group/events/past/

File: js/meetup/data.js [hunk 3]
```diff
@@ -1,4 +1,43 @@
...
+        slides_link: '/assets/meetup_sept_2020_minitest_victor.pdf',
+        video_link: '',
+        demo_link: ''
+      },
+      {
+        id: 2,
+        title: 'Introduction to Kubernetes',
+        description: null,
+        speaker: 'Dibesh Shrestha',
+        twitter_username: '',
```
Comment (zoras) [inline js/meetup/data.js:32]: https://twitter.com/dibeshs

File: js/meetup/data.js [hunk 4]
```diff
@@ -1,4 +1,43 @@
...
+        github_username: '',
```
Comment (zoras) [inline js/meetup/data.js:33]: https://github.com/dibeshs

File: js/meetup/data.js (hunk 1 above)
Comment (zoras) [reply js/meetup/data.js:36]: looks like this problem exists with some other twitter profiles as well. 
Let's reverse this and tackle it in separate PR.

**Review by zoras — CHANGES_REQUESTED** [review]
looks good but some changes needed

File: js/meetup/data.js (hunk 1 above)
Comment (zoras) [reply js/meetup/data.js:36]: Fixed it in https://github.com/RubyNepal/rubynepal.github.io/pull/424

File: js/meetup/data.js (hunk 2 above)
Comment (josisusan) [reply js/meetup/data.js]: Updated to 19

File: js/meetup/data.js (hunk 3 above)
Comment (josisusan) [reply js/meetup/data.js]: Added

File: js/meetup/data.js (hunk 4 above)
Comment (josisusan) [reply js/meetup/data.js]: Added

**Review by zoras — APPROVED** [review]
LGTM :100:

**Pull request merged**